exonum/src/proto/schema/exonum/binary_map.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Internal data types of Bitcoin anchoring service.
+
+syntax = "proto3";
+
+package exonum;
+
+// Some non-scalar key-value pair.
+message KeyValue {
+    bytes key = 1;
```

Comment:
Nit: We use 2-space indentation as per [Proto3 style guide](https://developers.google.com/protocol-buffers/docs/style).

Reply:
We can use clang-format to formatting proto files.